marketplace/settings.py

```diff
@@ -348,3 +348,5 @@
         "ENVIRONMENT": env("APM_SERVICE_ENVIRONMENT", default="production"),
         "DEBUG": env.bool("APM_SERVICE_DEBUG", default=False),
     }
+
+FLOWS_REST_ENDPOINT = env.str("FLOWS_REST_ENDPOINT")
```

Comment:
You can leave a blank line in the end of code. Can see more in: https://peps.python.org/pep-0008/#blank-lines